client/hotrod-client/src/test/java/org/infinispan/client/hotrod/evolution/model/BaseModelWithNameAnalyzedAndNameNonAnalyzedFieldEntity.java

```diff
@@ -22,19 +22,19 @@ public class BaseModelWithNameAnalyzedAndNameNonAnalyzedFieldEntity implements M
 
     @ProtoField(number = 3)
     @Deprecated(forRemoval=true)
-    @Text()
-    public String nameAnalyzed;
+    @Text
+    public String name;
 
     @ProtoField(number = 4)
     @Basic(projectable = true)
-    public String nameNonAnalyzed;
+    public String analyzed;
```

Comment:
I think it is not analyzed

Reply:
I know, but I was concerned mostly about fixing those classes' compatibility issues. They were renaming and renumbering fields, which is now no longer allowed. We can be more accurate in future work.